src/shipment/shipment.repository.ts

```diff
@@ -0,0 +1,60 @@
+import { Shipment } from './entities/shipment.entity';
+
+/**
+ * Repository class for storing shipments in memory.
+ * It is needed besides the queue to keep track of the shipments for manual updates.
+ */
+export class ShipmentRepository {
+  private shipments: Shipment[] = [];
+
+  /**
+   * Creates a new shipment and adds it to the repository.
+   * @param shipment - The shipment to be created.
+   * @returns The created shipment.
+   */
+  public create(shipment: Shipment): Shipment {
```

Comment:
Are there any checks elsewhere that prevent a shipment from being added multiple times?